Gemfile

```diff
@@ -1,25 +1,26 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.5.3'
+ruby '3.1.2'
 
-gem 'autoprefixer-rails'
-gem 'bootsnap', '>= 1.4.2', require: false
-gem 'bootstrap-sass', '~> 3.4.1'
-gem 'devise'
-gem 'faker', :git => 'https://github.com/stympy/faker.git', :branch => 'master'
+gem 'autoprefixer-rails' 
+gem 'bootsnap', '>= 1.4.2', require: false 
+gem 'bootstrap-sass', '~> 3.4.1' 
+gem 'devise' 
+gem 'faker', :git => 'https://github.com/stympy/faker.git', :branch => 'main'
```

Comment:
Lines 6 thru 10 are the same except for the change of 'master' to 'main' in line 10.
This change is necessary since the faker repo has changed and w/o it bundle install fails.